mapper/mapper.go

```diff
@@ -321,6 +323,42 @@ func mapFilters(page *model.SearchPage, categories []data.Category, queryParams
 	page.Data.Filters = filters
 }
 
+func mapTopicFilters(page *model.SearchPage, topicCategories []data.TopicCategory, queryParams data.SearchURLParams) {
```

Comment:
similar as previous comment; do we need to duplicate this functionality when it's doing the same/virtually the same?